docs/source/checks/tabular/model_evaluation/plot_weak_segments_performance.py

```diff
@@ -13,26 +13,28 @@
 * `Define a condition <#define-a-condition>`__
 
 What is the purpose of the check?
-=================================
+==================================
```

Comment:
Seems like this should be 
```suggestion
-----------------------------------
```
Otherwise we'd have access titles in the gallery menu no?

Reply:
I think i prefer it as it is now (a sub-title)